### PR TITLE
Ft es6modules

### PR DESCRIPTION
**Pull request by DjibrilM**
## In the following pull request I introduced 

- ES6 modules 
-  ES6 syntaxes and time outputting

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @DjibrilM 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors and no  future features section in readme, please add it 👍🏿 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: main.js [hunk 1]
```diff
@@ -1,41 +1,25 @@
...
 const form = document.querySelector('form');
 const booksListElement = document.querySelector('.listOfBooks');
 const navigationLinks = document.querySelectorAll('.nav-link');
 
-class Main {
+class Main extends BookElement {
   constructor() {
+    super();
     this.listOfBooks = [];
   }
```
Comment (Owusu-Desmond) [inline main.js]: - [ ] Please from the project requirement, you should create the entry point for your JavaScript code called `index.js` :blush: 

- [ ] From the project requirement, you should divide your code into reasonable modules. I think can create a module for this `Main` class in your main.js and import the module into the `main.js` which is supposed to be `index.js` as your app entry point. :relaxed:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @DjibrilM , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please make sure your app works are expected, currently, I can't add a book and the current date and time is not displaying.
  [Screencast from 06-12-2023 04:53:12 PM.webm](https://github.com/DjibrilM/Awesome-books/assets/92303891/a22d9c38-edc8-4a83-a8e8-ea81e1b0a28c)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 2]
```diff
@@ -87,11 +72,24 @@ class Main {
...
 }
 
 const mainClass = new Main();
 mainClass.formSubmit();
 mainClass.loadData();
+mainClass.getTime();
+
 navigationLinks.forEach((el) => {
   el.addEventListener('click', () => {
     document.querySelector('.active').classList.remove('active');
```
Comment (tresorsawasawa) [inline index.js:95]: - [ ] As mentioned by the previous reviewer and for the project requirements, please make sure your app is working as expected, currently, User can't `add a book` (and impossible to test the `remove` action as there is no added book), the` current date and time is not displaying`, and the `contact page doesn't work`. Make sure you fix those issues to match the expected result.
I recommend you first test all the features of your application and make sure everything is working before requesting a new review.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @DjibrilM and @AsumVictor 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DjibrilM** [comment]
https://github.com/DjibrilM/Awesome-books/assets/82331014/e9dd24c7-4883-4486-a80a-5622c122d2cd

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @DjibrilM ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed GitHub Flow ✔️
- Linters are passing ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**